sklearn/linear_model/_logistic.py

```diff
@@ -1884,31 +1938,64 @@ def fit(self, X, y, sample_weight=None, **params):
         """
         _raise_for_params(params, self, "fit")
 
-        solver = _check_solver(self.solver, self.penalty, self.dual)
+        if isinstance(self.l1_ratios, str) and self.l1_ratios == "warn":
```

Comment:
Why warn by default? I would expect `LogisticRegressionCV(Cs=np.logspace(-6, 6, 13)).fit(X_train, y_train)` to behave the same in 1.10 and 1.8, and so I don't see the point in warning the user when using the default.

I think we should only warn when the user explicitly passes `LogisticRegressionCV(Cs=np.logspace(-6, 6, 13), l1_ratios=None).fit(X_train, y_train)` instead.

Reply:
In that case, same as `l1_ratio` for LogisticRegression. I will add the changes accordingly.